tests/rados/monitor_configurations.py

```diff
@@ -239,6 +239,7 @@ def remove_config(self, **kwargs):
                 1. section: which section of daemons to target
                     allowed values: global, mon, mgr, osd, mds, client
                 2. name: name of the config param for the selection
+                3. verify_removal: (bool) specifies if the removal should be verified
```

Comment:
the keyword argument variable is `verify_rm`, so doc-text should also describe `verify_rm` instead of `verify_removal`

Reply:
Updated.